src/pages/workspace/expensifyCard/WorkspaceCardsListLabel.tsx

```diff
@@ -0,0 +1,125 @@
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useResponsiveLayout from '@hooks/useResponsiveLayout';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import * as CurrencyUtils from '@libs/CurrencyUtils';
+import getClickedTargetLocation from '@libs/getClickedTargetLocation';
+import variables from '@styles/variables';
+import * as Report from '@userActions/Report';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+type WorkspaceCardsListLabelProps = {
+    /** Label type */
+    type: ValueOf<typeof CONST.WORKSPACE_CARDS_LIST_LABEL_TYPE>;
+
+    /** Label value */
+    value: number;
+
+    /** Additional style props */
+    style?: StyleProp<ViewStyle>;
+};
+
+function WorkspaceCardsListLabel({type, value, style}: WorkspaceCardsListLabelProps) {
+    const styles = useThemeStyles();
+    const {windowWidth} = useWindowDimensions();
+    const {shouldUseNarrowLayout} = useResponsiveLayout();
+    const theme = useTheme();
+    const {translate} = useLocalize();
+    const [bankAccountList] = useOnyx(ONYXKEYS.BANK_ACCOUNT_LIST);
+    const [isVisible, setVisible] = useState(false);
+    const [anchorPosition, setAnchorPosition] = useState({top: 0, left: 0});
+    const anchorRef = useRef(null);
+
+    const isConnectedWithPlaid = useMemo(() => !!Object.values(bankAccountList ?? {})[0]?.accountData?.additionalData?.plaidAccountID, [bankAccountList]);
+
+    useEffect(() => {
+        if (!anchorRef.current || !isVisible) {
+            return;
+        }
+
+        const position = getClickedTargetLocation(anchorRef.current);
+        const BOTTOM_MARGIN_OFFSET = 3;
+
+        setAnchorPosition({
+            top: position.top + position.height + BOTTOM_MARGIN_OFFSET,
+            left: position.left,
+        });
+    }, [isVisible, windowWidth]);
+
+    const requestLimitIncrease = () => {
+        // TODO: uncomment when RequestExpensifyCardLimitIncrease API call is supported
+        // Policy.requestExpensifyCardLimitIncrease(settlementBankAccountID);
+        setVisible(false);
+        Report.navigateToConciergeChat();
+    };
+
+    return (
+        <View style={styles.flex1}>
+            <View
+                ref={anchorRef}
+                style={[styles.flexRow, styles.alignItemsCenter, styles.mb1, style]}
+            >
+                <Text style={[styles.mutedNormalTextLabel, styles.mr1]}>{translate(`workspace.expensifyCard.${type}`)}</Text>
+                <PressableWithFeedback
+                    accessibilityLabel={translate(`workspace.expensifyCard.${type}`)}
+                    accessibilityRole={CONST.ROLE.BUTTON}
+                    onPress={() => setVisible(true)}
+                >
+                    <Icon
+                        src={Expensicons.Info}
+                        width={variables.fontSizeLabel}
```

Comment:
I think this is 13, which is not the correct value we'd want to use. We should have some existing icon size variables you can use instead - can you take a look?

Reply:
Perfect, we have these at 12x12 in Figma so let's do that here too.